components/bms_worker/include/BatteryMonitoring.h

```diff
@@ -71,25 +71,25 @@ typedef struct
     bool        fault;
     uint16_t    balancing_cells;
     BMS_Cell_S  cells[MAX_CELL_COUNT];      // [mv], precision 1mv
-    uint16_t    pack_voltage;               // [mv], precision 1mv
-    uint16_t    calculated_pack_voltage;    // [mv], precision 1mv
+    float32_t   pack_voltage;               // [mv], precision 1mv
+    float32_t   calculated_pack_voltage;    // [mv], precision 1mv
     uint8_t     connected_cells;
 
-    float charge_limit;
-    float discharge_limit;
+    float32_t charge_limit;
+    float32_t discharge_limit;
 
     struct
     {
-        uint16_t max;
-        uint16_t min;
-        uint16_t avg;
-    } voltage;    // [0.1mv], precision 0.1mv
+        float32_t max;
+        float32_t min;
+        float32_t avg;
```

Comment:
unrelated to your changes, but you guys should make a type for this. It's a pretty common pattern.

```c
typedef struct
{
    float32_t max;
    float32_t min;
    float32_t avg;
} MinMaxAvg_S;
``` 
or something to that effect